src/main/java/gov/nih/nci/evs/api/model/Terminology.java

```diff
@@ -36,6 +36,9 @@
 
   /** The graph. */
   private String graph;
+  
+  /** The graph source. */
+  private String source;
```

Comment:
Added this new field in `Terminology` to use the query. Since we are already using the `graph` for the name of the graph we provide, I called this source. The prefix does not work with graph name and requires the top level subject to be specified.